packages/react-reconciler/src/__tests__/ReactDeferredValue-test.js

```diff
@@ -744,6 +744,7 @@ describe('ReactDeferredValue', () => {
         </Container>,
       );
       // We should switch to pre-rendering the new preview.
+      await waitForPaint([]);
```

Comment:
This happens because the `requestPaint()` is now unconditional even if there weren't any effects. Which doesn't really hurt anything since it's actually a noop in anything but tests and should probably go away.